### PR TITLE
Cypress test. Create task with files from remote sources.

### PR DESCRIPTION
**Pull request by dvkruchinin**
### Motivation and context
Add Cypress test.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: tests/cypress/integration/actions_tasks_objects/case_46_create_task_with_files_from_remote_sources.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            cy.deleteTask(taskName, $taskID);
+        });
+    });
+
+    describe(`Testing "${labelName}"`, () => {
+        it('Try to create a task with wrong remote file. The task is not created.', () => {
+            cy.get('[id="name"]').type(taskName);
+            cy.addNewLabel(labelName);
+            cy.contains('Remote sources').click();
+            cy.get('[placeholder="Enter one URL per line"]').type(wrongUrl);
```
[the remark this answers is not in the document]
Comment (bsekachev) [reply tests/cypress/integration/actions_tasks_objects/case_46_create_task_with_files_from_remote_sources.js:33]: Let's say: ``cvat-file-selector-remote``

File: tests/cypress/integration/actions_tasks_objects/case_46_create_task_with_files_from_remote_sources.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            cy.contains('[type="button"]', 'Submit').click();
+            cy.get('.cvat-notification-notice-create-task-failed').should('exist');
+            cy.closeNotification('.cvat-notification-notice-create-task-failed');
+        });
+
+        it('Set correct URL to remote file. The task is created.', () => {
+            cy.get('[placeholder="Enter one URL per line"]').clear().type(correctUrl);
```
Comment (bsekachev) [inline tests/cypress/integration/actions_tasks_objects/case_46_create_task_with_files_from_remote_sources.js:40]: The same

File: tests/cypress/integration/actions_tasks_objects/case_46_create_task_with_files_from_remote_sources.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+context('Create a task with files from remote sources.', () => {
+    const caseId = '46';
+    const labelName = `Case ${caseId}`;
+    const taskName = `New annotation task for ${labelName}`;
+    const wrongUrl =
+        'https://raw.githubusercontent.com/openvinotoolkit/cvat/develop/cvat/apps/documentation/static/documentation/images/cvatt.jpg';
```
[the remark this answers is not in the document]
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/case_46_create_task_with_files_from_remote_sources.js]: Thanks. Replaced it.

File: tests/cypress/integration/actions_tasks_objects/case_46_create_task_with_files_from_remote_sources.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/case_46_create_task_with_files_from_remote_sources.js]: Added css class. Сhanged it in the test.

File: tests/cypress/integration/actions_tasks_objects/case_46_create_task_with_files_from_remote_sources.js (hunk 2 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/case_46_create_task_with_files_from_remote_sources.js]: Added css class. Сhanged it in the test.

**Review by bsekachev — APPROVED** [review]

**Pull request merged**